### PR TITLE
updated apt cache sensu check

### PR DESCRIPTION
**Pull request by pgraziano**

### CODE REVIEW

File: sensu/plugins/check-apt-cache.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+WARNING_HOURS=$1
+CRITICAL_HOURS=$2
+
+if [ $# -ne 2 ]; then
+    echo "usage: $0 WARNINGHOURS CRITICALHOURS"
+    exit 3
+fi
+
+if ! [[ -f $FILE ]]; then
+    echo "status file missing: $FILE"
```
Comment (scoop206) [inline sensu/plugins/check-apt-cache.sh]: would it make sense have this part be idempotent, e.g. just create the file if it isn't there ?

Comment (pgraziano) [reply sensu/plugins/check-apt-cache.sh]: no, because this file is created/touched by the cron job that runs `apt-get update` on success

Comment (scoop206) [reply sensu/plugins/check-apt-cache.sh]: oh I see, `update-cache-success` is built in sentinel for apt ?

Comment (pgraziano) [reply sensu/plugins/check-apt-cache.sh]: yes, I lay that down in ursula here https://github.com/blueboxgroup/ursula/pull/2993/files#diff-e39d5f40d5081a492e843ede007c8742R15

File: sensu/plugins/check-apt-cache.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    exit 3
+fi
+
+HOURS_SINCE_UPDATE=`echo $(( ($(date +%s) - $(date +%s -r "$FILE"))/60/60))`
+MINUTES_SINCE_UPDATE=`echo $(( ($(date +%s) - $(date +%s -r "$FILE"))/60))`
+
+if [[ $HOURS_SINCE_UPDATE -gt $WARNING_HOURS ]]; then
+    echo "WARNING: stale apt cache, $HOURS_SINCE_UPDATE hours old."
+    exit 1
+elif [[ $HOURS_SINCE_UPDATE -gt $CRITICAL_HOURS ]]; then
```
Comment (scoop206) [inline sensu/plugins/check-apt-cache.sh]: I think we need check CRIT first, otherwise it will always only warn ?

Comment (pgraziano) [reply sensu/plugins/check-apt-cache.sh]: good catch, just fixed it

**Review by scoop206 — APPROVED** [review]

**Pull request merged**